v3/src/components/case-tile-common/attribute-menu/attribute-menu.scss

```diff
@@ -20,8 +22,198 @@
 }
 
 // Formula Editor Styles
+.formula-modal-body {
+  cursor: default;
+}
+
+.formula-modal-footer {
+  cursor: default;
+}
 .formula-form-control {
   input {
     max-width: 270px;
   }
 }
+
+.formula-editor-button {
+  padding: 0 10px;
+  &:hover {
+    background: $codap-teal-lighter !important;
+  }
+  &:active, &.menu-open {
+    background: $codap-teal-light !important;
```

Comment:
I didn't check to make sure it's possible, but we should really avoid `!important` whenever we can. It can cause a lot of confusion and frustration down the line.

Reply:
These were to ChakraUI's styling